Automattic/woocommerce-payments
# Redirect to Stripe KYC to complete progressive onboarding to enable payouts

Fixes #4812 

#### Changes proposed in this Pull Request

Tweak the task list (Things To Do) by supporting completing the second step of KYC to enable payouts for merchants.

#### Testing instructions

* Checkout to server branch `dev/ask-for-eventually-due`.
* Complete the progressive onboarding, check the Stripe account that it has payments enabled and payouts disabled.
* Find `Update WooCommerce Payments business details` block and click `Update` button that should redirect to the Stripe KYC asking for business details and bank account.
* Verify that it allowed payouts on the same Stripe account (need to wait a few mins)

#### Screenshots
<img width="500" alt="image" src="https://user-images.githubusercontent.com/79862886/199720109-7641bee8-d877-42a9-b806-1a971b0a9a2b.png">

<img width="500" alt="image" src="https://user-images.githubusercontent.com/79862886/194130336-82d8704d-1530-4e17-86df-950ca11220f7.png">

<img width="500" alt="image" src="https://user-images.githubusercontent.com/79862886/194128822-c608f4ad-1a1e-4cf9-9078-bf939c4ae1b3.png">
<img width="500" alt="image" src="https://user-images.githubusercontent.com/79862886/194128884-bd4e93a7-4581-4871-9c96-a022e853f946.png">
<img width="500" alt="image" src="https://user-images.githubusercontent.com/79862886/194129331-12f8cc3c-e039-46e0-a8f3-08f4aa7391d3.png">

-------------------


**Post merge**
